README.md

```diff
@@ -61,6 +61,18 @@ across the V8 and Test262 projects.
   $ gits push origin migration-twenty-three
   ```
 
+## Runtimes
+
+- [io.js](https://iojs.org/en/index.html)
+  + `brew install iojs`
```

Comment:
I don't know if it's worth including Homebrew instructions, if we don't offer instructions for any other platforms. You might say, "it couldn't hurt", but I don't want to give the impression that this is a OSX-only project.